build/docker/zap-full-scan.py

```diff
@@ -85,6 +85,7 @@ def usage():
     print('    -r report_html    file to write the full ZAP HTML report')
     print('    -w report_md      file to write the full ZAP Wiki(Markdown) report')
     print('    -x report_xml     file to write the full ZAP XML report')
+    print('    -J report_json    file to write the full ZAP JSON document')
```

Comment:
Fixed in zap-full-scan.py, can probably patch the other scripts as well if it's about as trivial of an update